resources/setups/baseline/module_config.json

```diff
@@ -1,10 +1,13 @@
 {
   "modules": {
     "ipaddr": {
-      "enabled": false,
+      "enabled": true,
       "timeout_sec": 900,
       "port_flap_timeout_sec": 20,
-      "dhcp_ranges": [{"start": "192.168.0.1", "end": "192.168.255.254", "prefix_length": 16}]
+      "dhcp_ranges": [{"start": "192.168.0.1", "end": "192.168.255.254", "prefix_length": 16},{"start": "10.0.0.1", "end": "10.255.255.254", "prefix_length": 8},{"start": "172.16.0.1", "end": "172.31.255.254", "prefix_length": 12}]
```

Comment:
can we format this a bit nicer so it makes a good example?  All the dhcp_ranges json on one line is very ugly!

Reply:
See commit c7cff772639e46112af4a7daff51ad0539e291af